bot/exts/filtering/_ui/filter.py

```diff
@@ -387,7 +388,7 @@ def description_and_settings_converter(
     if not SINGLE_SETTING_PATTERN.match(parsed[0]):
         description, *parsed = parsed
 
-    settings = {setting: value for setting, value in [part.split("=", maxsplit=1) for part in parsed]}
+    settings = dict([part.split("=", maxsplit=1) for part in parsed])
```

Comment:
This is undesirable here since the variable name helps keep track of the code

Reply:
https://github.com/python-discord/bot/pull/2523/commits/33f4a76ca74d58201a582d4dd838d142eed4d6fb